ccp/apis/v1/ccp_server/backend/builds/build_logs.py

```diff
@@ -1,6 +1,77 @@
 # Process response for /$namespace/$app_id/$job_id/$desired-tag/$build/build-logs
+from ccp.apis.v1.ccp_server import meta_obj
+from ccp.lib.processors.pipeline_information.builds import \
+    OpenshiftJenkinsBuildInfo
+from ccp.apis.v1.ccp_server.models.build_logs import BuildLogs
+from ccp.apis.v1.ccp_server.models.prebuild_lint_build_scan_logs import \
+    PrebuildLintBuildScanLogs
+from ccp.apis.v1.ccp_server.models.scanner_logs import ScannerLogs
+from ccp.apis.v1.ccp_server.models.all_scanner_logs import AllScannerLogs
+from ccp.index_reader import Project
+from ccp.apis.v1.ccp_server.env_config import *
 
-def response():
+from typing import List, Dict  # noqa: F401
+
+
+def process_log(log_obj, stage):
+    logs = None
+    if log_obj and len(log_obj.keys()) > 0:
+        for k, v in log_obj:
+            if k == stage:
+                logs = "{}<br />\n{}".format(
+                    k,
+                    v
+                )
+                break
+    return logs
+
+
+def response(namespace, appid, jobid, desired_tag, build):
     """
     """
-    return {}
+    ojbi = OpenshiftJenkinsBuildInfo(
+        JENKINS_URL,
+        token_from_mount=SERVICE_ACCOUNT_SECRET_MOUNT_PATH,
+        namespace=namespace
+    )
+
+    job_name = Project.pipeline_name(
+        app_id=appid, job_id=jobid, desired_tag=desired_tag
+    )
+    jenkins_job_name = "{}-{}".format(
+        namespace,
+        job_name
+    )
+    logs_info =ojbi.get_build_logs(
+        ordered_job_list=[
+            namespace,
+            jenkins_job_name
+        ],
+        build_number=build
+    )
+    prebuild_logs = process_log(logs_info, "Prebuild source repo")
+    prebuild_exists = True if prebuild_logs else False
```

Comment:
if there is any issue in retrieving the prebuild logs. we will fail to know if there was any prebuild requested or not. we need to first check for prebuild exists or not then pull log. because we print `Prebuild not requested` as log if the prebuild stage is not requested.